src/Tests/Databricks.fs

```diff
@@ -0,0 +1,53 @@
+module Databricks
+
+open Expecto
+open Farmer
+open Farmer.Builders
+open Farmer.Databricks
+open System
+
+let tests = testList "Databricks Tests" [
```

Comment:
I think these tests could (should) go down to the ARM level at least i.e. you can upcast `workspace` to an `IBuilder` and then call `buildResources` to get the raw ARM output. I've modified the first test to illustrate this.

There's a final test we should add to which takes the *generated ARM* json into a big file and is used for regression testing - let's not forget about that, we can do it at the end.

Reply:
This doesn't seem to compile for me. It gives me the error: `Type constraint mismatch. The type 'unit' is not compatible with type 'IBuilder'`

Reply:
Yeah, looks like an Isaac special - broke the build.

Reply:
It's the `Run()` method - it needs to return the `state` back out if the validation passes. I'll fix it up.